boards/arm/verdin_imx8mm_m4/Kconfig.defconfig

```diff
@@ -0,0 +1,18 @@
+# VERDIN_IMX8MM_M4 board defconfig
+
+# Copyright (c) 2023, Toradex
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_VERDIN_IMX8MM_M4
+
+config BOARD
+	default "verdin_imx8mm_m4"
+
+if !XIP
```

Comment:
Why include this, if `CONFIG_XIP` is enabled by default in the board defconfig?

Reply:
Fixed, I actually based this one on the imx8mm evk from NXP. thanks